wiki/About/id.md

```diff
@@ -0,0 +1,6 @@
+Tentang osu!wiki
+=======
+
+osu!wiki adalah sebuah proyek yang dibuat dan dijalankan oleh komunitas yang bertujuan untuk membantu para pemain osu! memahami permainan secara lebih efektif.
```

Comment:
Also you can use 
> memahami permainan ini lebih baik.

Reply:
Kayaknya aneh nanti deh kalo pake kalimat itu.